protocol-tests/src/test/java/fixtures/dpgcustomization/DpgCustomizationTests.java

```diff
@@ -213,4 +308,20 @@ public void testSendRequestMethod() {
         Assertions.assertTrue(rawModel.containsKey("received"));
         Assertions.assertEquals("raw", rawModel.get("received"));
     }
+
+    static class TracingValidationPolicy implements HttpPipelinePolicy {
+
+        public HttpPipelineCallContext context;
+
+        TracingValidationPolicy() {
+        }
+
+        @Override
+        public Mono<HttpResponse> process(HttpPipelineCallContext httpPipelineCallContext, HttpPipelineNextPolicy httpPipelineNextPolicy) {
+            this.context = httpPipelineCallContext;
```

Comment:
It should validate that every "context" has certain key, not just log the last "context".